paasta_tools/automatic_rollbacks/slack.py

```diff
@@ -69,6 +56,33 @@ def get_slack_blocks_for_deployment(
             },
         },
     ]
+    if slo_watchers is not None and len(slo_watchers) > 0:
+        num_failing = len([w for w in slo_watchers if w.failing])
```

Comment:
eventually I would like this to be a function where we can make more elaborate messages and stuff